arangod/Cluster/ClusterInfo.cpp

```diff
@@ -2416,7 +2418,7 @@ Result ClusterInfo::createCollectionsCoordinator(
   // closure and the main thread executing this function. Note that it can
   // happen that the callback is called only after we return from this
   // function!
-  auto dbServerResult = std::make_shared<std::atomic<int>>(-1);
+  auto dbServerResult = std::make_shared<std::atomic<std::optional<ErrorCode>>>();
```

Comment:
This is slightly inconsistent (though not a problem):
The similar changes a few lines above initialize the result with an `std::nullopt`, but this place doesn't. Feel free to leave it as it is.

Reply:
Thanks, fixed.